src/Symfony/Component/DependencyInjection/Tests/Fixtures/config/nested_bundle_config.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use Symfony\Config\AcmeConfig\NestedConfig;
+
+return static function (NestedConfig $config) {
+    throw new RuntimeException('This code should not be run.');
```

Comment:
We dont actually need this line, right?

Reply:
I think this code is need to prevent implicit change an actual logic.

Reply:
Could you elaborate?

Reply:
If this function will be executed (for some unknown reason to me), this exception may help figure out what is happened faster.

Reply:
Sure. Okey.